src/cli/cli.py

```diff
@@ -142,6 +145,18 @@ def stop(self, cmd: str) -> None:
             return
         self.container_manager.stop(cmd)
 
+    def kill(self, cmd: str) -> None:
```

Comment:
What situation is this needed for?

Reply:
If the SSH connection to the container stops functioning for some reason. 
It happened to me once. I was not able to replicate it or find what caused it.
I think it's useful as a failsafe.